### ISSUE TITLE
Document the bulk actions on slow connections

### ISSUE THREAD

**Issue by pixdemo2** [KeypDemoCorp/cooking-app-frontend#189]
There is a temporary workaround in place that we should remove afterwards.

Accessibility should be considered when implementing this.
